### PR TITLE
fix(rdb): import privileges and update documentation

### LINKED ISSUE #1718: RDB privilege can not import
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Affected Resource(s)

* scaleway_XXXX

### Terraform Configuration Files

```hcl
# Copy-paste the minimal (if possible) terraform configuration to reproduce the bug here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key: https://keybase.io/hashicorp
```

### Steps to Reproduce

1. `terraform apply`

### References

* #0000

### PR DESCRIPTION
**Pull request by Monitob**
close #1718

### CODE REVIEW

File: docs/data-sources/rdb_privilege.md [hunk 1]
```diff
@@ -30,6 +31,9 @@ data "scaleway_rdb_privilege" "find_priv" {
 
 ## Attributes Reference
 
-In addition to all arguments above, the following attributes are exported:
+In addition to all above arguments, the following attribute is exported:
```
Comment (quantumsheep) [inline docs/data-sources/rdb_privilege.md:34]:
```suggestion
In addition to all above arguments, the following attributes are exported:
```

**Review by quantumsheep — CHANGES_REQUESTED** [review]

File: docs/data-sources/rdb_privilege.md [hunk 2]
```diff
@@ -30,6 +31,9 @@ data "scaleway_rdb_privilege" "find_priv" {
 
 ## Attributes Reference
 
-In addition to all arguments above, the following attributes are exported:
+In addition to all above arguments, the following attributes are exported:
 
-- `permission` - The permission for this user on the database. Possible values are `readonly`, `readwrite`, `all`, `custom` and `none`.
+- `id` - The resource ID
+- `region` - The Scaleway region this bucket resides in.
```
Comment (Codelax) [inline docs/data-sources/rdb_privilege.md:37]: `bucket`

**Review by quantumsheep — DISMISSED** [review]

**Review by quantumsheep — DISMISSED** [review]

File: scaleway/data_source_rdb_privilege.go [hunk 3]
```diff
@@ -20,8 +20,13 @@ func dataSourceScalewayRDBPrivilege() *schema.Resource {
 }
 
 func dataSourceScalewayRDBPrivilegeRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
-	instanceID, _ := d.Get("instance_id").(string)
+	region, instanceID, err := parseRegionalID(d.Get("instance_id").(string))
+	if err != nil {
+		return diag.FromErr(err)
+	}
```
Comment (Codelax) [inline scaleway/data_source_rdb_privilege.go:26]: Does this work with
```hcl
data "scaleway_rdb_privilege" "main" {
  region = "fr-par"
  instance_id   = "11111111-1111-111111111111"
  user_name     = "my-user"
  database_name = "my-database"
}
```

Is regional ID required ? It should not be

Comment (Codelax) [reply scaleway/data_source_rdb_privilege.go:26]: If fixed, data source example in doc should also be fixed.
We should avoid locality IDs in examples
https://github.com/scaleway/terraform-provider-scaleway/pull/1721/files#diff-c620de89f9ac495bb7a44a54b2223dd12308a81aefccf29ed9f1420f12613e1aR17-R21

**Review by Codelax — CHANGES_REQUESTED** [review]

File: scaleway/data_source_rdb_privilege.go (hunk 3 above)
Comment (Monitob) [reply scaleway/data_source_rdb_privilege.go]: Thanks for the review. Many changes on the last commit in order to handle the migration on schema.

**Review by quantumsheep — DISMISSED** [review]

**Review by Codelax — APPROVED** [review]

**Pull request merged**